components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/deployment/AppDeployerServiceComponent.java

```diff
@@ -65,6 +69,23 @@ protected void activate(ComponentContext ctxt) {
         CAppDeploymentManager cAppDeploymentManager = new CAppDeploymentManager(configCtx.getAxisConfiguration());
         initializeDeployers(artifactDeploymentManager, cAppDeploymentManager);
 
+        // Register eventSink deployer
+        try {
+            String carbonRepoPath = configCtx.getAxisConfiguration().getRepository().getPath();
+            String eventSinkPath = carbonRepoPath + File.separator + "event-sinks";
+            Class deployerClass = Class.forName("org.wso2.micro.integrator.event.sink.EventSinkDeployer");
+            Deployer deployer = (Deployer) deployerClass.newInstance();
+            artifactDeploymentManager.registerDeployer(eventSinkPath, deployer);
```

Comment:
Shall we add some info or debug logs here saying that this deployer is registered successfully.

Reply:
Done in bd10bd3